Automattic/vip-go-mu-plugins
# Update ElasticPress submodule to 8e9c061e

## Description

Update ElasticPress fork to 8e9c061e

## Changelog Description

### Search: updated dependencies

We've implemented additional logging for Elasticsearch queries to provide the stack trace of where the query has been called from.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Make sure `EP_QUERY_LOG` is set to `true`
1. in `wp shell` initialize a `new WP_Query( [ 'es' = true, 'post_type' => ['post'] ] );`
1. Call `ep_get_query_log` and verify the query has `backtrace` property that contains an array of the backtrace.
